scripts/prepare_slave_container_buildinfo.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+SLAVE_DIR=$1
+
+BUILDINFO=$SLAVE_DIR/buildinfo
+BUILD_VERSIONS_PATH=$SLAVE_DIR/buildinfo/build/versions
+VERSION_DEB_PREFERENCE=$BUILD_VERSIONS_PATH/01-versions-deb
+
+# Enable the build hooks
+set_build_hooks
```

Comment:
Better name: `symlink_hooks`

Reply:
Changed to symlink_build_hooks.

---
In reply to: [539817491](https://github.com/Azure/sonic-buildimage/pull/5718#discussion_r539817491) [](ancestors = 539817491)